src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -477,14 +477,11 @@ public WebElement swipeTo(SwipeElementDirection s, By by, int attempts) {
             return driver.findElement(by);
         } else {
             for (i = 0; i < attempts; i++) {
-
-                swipeCenterLong(s);
-                if (isPresentWait(by)) {
+                swipeCenter(s);
+                if (isPresent(by)) {
```

Comment:
Yeah this would be the one drawback. My thinking was that since it initially does an isPresentWait before swiping that it would give the element time to load. So far we have had luck with using this to find a webview element loading. So I think think might work well until we find a reason to change it back.

Reply:
ah I just realized what you mean by asynchronously. I think the webview might not do that actually. I'll try some more tests

Reply:
even if it was a news feed that takes a few to load, would the scroll be so long that it wouldn't catch it on the next scroll?

Reply:
I would approve this change for now but use a shorter wait than the default "isPresentWait" as they're serving slightly different purposes (IMO).